### PR TITLE
TAJO-1887: Disable the alter table add partition statement temporarily.

### PR DESCRIPTION
**Pull request by blrunner**
Currently, users can use arbitary partition path using alter table statement. But they can't scan their partitions because PartitionedTableRewriter just build tuple from partition path. Fundamentally, we should allow arbitary partition path. But we need to avoid an exception by above situation until tajo allow arbitary partition path. So it seems that alter table add partition statement is disabled temporarily.

### CODE REVIEW

**Comment by hyunsik** [comment]
Rather than commenting out for disabling the feature, I'd like to recommend to add some verification rule.

**Comment by blrunner** [comment]
Thanks @hyunsik 

I reflected your comments. :)

File: tajo-plan/src/main/java/org/apache/tajo/plan/verifier/PreLogicalPlanVerifier.java [hunk 1]
```diff
@@ -354,4 +354,14 @@ public Expr visitInsert(Context context, Stack<Expr> stack, Insert expr) throws
 
     return expr;
   }
+
+  // TODO: This should be removed at TAJO-1891
+  @Override
+  public Expr visitAlterTable(Context context, Stack<Expr> stack, AlterTable expr) throws TajoException {
+    if (expr.getAlterTableOpType() == AlterTableOpType.ADD_PARTITION) {
+      context.state.addVerification(new UnsupportedException("ADD PARTITION"));
```
Comment (hyunsik) [inline tajo-plan/src/main/java/org/apache/tajo/plan/verifier/PreLogicalPlanVerifier.java]: It should be UnimplementedException. UnsupportedException means there is no plan to support this feature.

**Comment by blrunner** [comment]
Thanks @hyunsik 
I've reflected your comments just now. :)

**Comment by hyunsik** [comment]
Could you add negative tests for ``ALTER ADD PARTITION``?

**Comment by blrunner** [comment]
I added negative unit test cases.

**Comment by hyunsik** [comment]
Could you rebase it against the latest revision?

**Comment by blrunner** [comment]
Rebased and updated the document for alter table statement.

**Comment by hyunsik** [comment]
+1
@blrunner thank you for your contribution. The patch looks good to me.

**Pull request closed without merging**